EIPS/eip-1328.md

```diff
@@ -7,12 +7,11 @@ category: ERC
 status: Draft
 created: 2018-08-15
 discussions-to: https://ethereum-magicians.org/t/wallet-connect-eip/850
-requires: 831
 ---
 
 ## Simple Summary
 
-A standard to create WalletConnect URIs for establishing connections between wallets and applications.
+A standard to create WalletConnect URIs for requuest connections between applications and wallets.
```

Comment:
typo - requuest - double u
perhaps in this change also to "to request" - or even "to initiate"

Reply:
> found typo and I think it could make sense to do this as a new EIP instead of changing the old one. When somewhere is written "EIP1328 supported" this would then not be true anymore as the - I think it is easier just to add it as a new EIP to prevent confusion in the future.
> Otherwise great - like that it got simper - subtraction FTW ;-)

I really don't want to have a duplicate standard when the previous one was barely adopted... I want to have WalletConnect backwards compatible starting from v1.0.0 and don't want to confuse new comers with an old architecture that is not going to be used at all.

As soon as all wallets move to v1.0.0 in the next month, there won't be anyone left using the legacy standard and it would just be polluting the EIPs